.cirrus.yml

```diff
@@ -0,0 +1,70 @@
+env:
+  intermediateArtifactName: dotnet-$CIRRUS_BUILD_ID.tar.gz
+
+Linux_task:
+  container:
+    image: ubuntu:xenial
+
+  packages_script: |
+    apt update
+    apt install -y libunwind8 libicu55 curl git clang-3.9 cmake make libc6-dev libssl-dev \
+                   libkrb5-dev libcurl4-openssl-dev zlib1g-dev liblldb-3.9-dev gettext \
+                   libicu-dev liblttng-ust-dev libkrb5-dev git
+
+  checkout repos_script: ./scripts/checkout.sh
+
+  build_script: |
+    cd $CIRRUS_WORKING_DIR/artifacts/sources/coreclr
+    bash -x ./build.sh -skiptests -skipmanagedtools -skipnative  -skipcrossgen -release -osgroup FreeBSD \
+        /p:osGroup=FreeBSD /p:PackageRid=freebsd-x64
+    cd $CIRRUS_WORKING_DIR/artifacts/sources/corefx
+    ./build.sh -c Release --restore
+#    ./build.sh -c Release --build  /p:osGroup=FreeBSD /p:PackageRid=freebsd-x64
+
+  publish_script: |
+    tar -cvzf $intermediateArtifactName $CIRRUS_WORKING_DIR/artifacts/sources/coreclr/bin # $CIRRUS_WORKING_DIR/artifacts/sources/corefx/bin
+    curl -s -X POST --data-binary @$intermediateArtifactName http://$CIRRUS_HTTP_CACHE_HOST/$intermediateArtifactName
+
+FreeBSD_task:
+  freebsd_instance:
+    image_family: freebsd-12-0
+
+  depends_on: Linux
+
+  env:
+    HOME: /root
+    GH_API_TOKEN: ENCRYPTED[18bd1bb47022f0ec649d768dd4b822a44ed5e0e378ea7f8703b2a55cd6535ab90a50e915d09fe48eae451bdb9b63aeae]
```

Comment:
@wfurt, this token is for publishing release artifacts. Here are all the steps I used:

1. Allowed CirrusCI to access my `freebsd-bootstrap-cli` fork: https://github.com/marketplace/cirrus-ci
2. Created a 'personal access token' with just one scope checkbox checked: `public_repo` https://github.com/settings/tokens <br/><img src="https://user-images.githubusercontent.com/3840695/52527184-0d072280-2ccd-11e9-9520-848a691efb9f.png" alt="screenshot-1" width="42%" />
3. Logged on to CirrucCI using GH account and encrypt token using `Secured Variables` facility provided at the setting page: <br/> <img src="https://user-images.githubusercontent.com/3840695/73611372-d915c300-45e9-11ea-9a06-5ccb502b700c.png" width="42%" alt="screenshot-2" />

Reply:
@wfurt, could you set it up so repo page show up on CirrusCI? https://cirrus-ci.com/github/wfurt/freebsd-bootstrap-cli